Makefile

```diff
@@ -71,16 +74,35 @@ FIND_CONFIG_SOURCE=$(shell find $(1) -name 'configure' -o -name 'CMakeLists.txt'
 # having to update this URL.
 GUEST_TOOLS_BINARIES_URL=https://github.com/S2E/s2e/releases/download/v2.0.0/
 
+SYSTEM_LLVM?=no
+
+ifneq ($(SYSTEM_LLVM), no)
+
+ifeq (, $(shell which llvm-config-$(SYSTEM_LLVM)))
+ $(error "No llvm-config-$(SYSTEM_LLVM) found. Please install llvm $(SYSTEM_LLVM) or add the llvm-config-$(SYSTEM_LLVM) binary to your path")
```

Comment:
Make sure that this links against static libraries, so that there is no need to do LD path tricks in other repos.

Reply:
Pushed a solution that should force static linking